environment.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - geopy
   - humanfriendly
   - importlib_resources
-  - iris>=3.2.1
+  - iris>=3.4.0
```

Comment:
are we sure we want to be **this** restrictive about the iris version? :beer:

Reply:
Yes, we do, this change was only introduced in iris 3.4 (see number 10 here: https://scitools-iris.readthedocs.io/en/latest/whatsnew/3.4.html#bugs-fixed). I will need iris 3.4 for the ICON to UGRID stuff (#1653) anyway...

Reply:
mmm ok - given that I would be very happy to pin to the iris version that will have the fix for netCDF4>1.6.0 issue, I can understand this one too :+1: I'd still want to hear what the techy folks think, but by all means - fine by me - @ESMValGroup/technical-lead-development-team are we ready to start pinning and using only the latest versions of iris?

Reply:
Looks like no objections.